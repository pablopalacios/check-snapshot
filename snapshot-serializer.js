const iWasCalled = () => {
  return new Error("I was called");
};

module.exports = {
  test(val) {
    throw iWasCalled();
  },
  iWasCalled
};
