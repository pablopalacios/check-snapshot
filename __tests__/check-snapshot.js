import React from "react";
import { iWasCalled } from "../snapshot-serializer";

import checkSnapshot from "../";

const FixtureComponent = () => <p>I'm a fixture component</p>;

describe("check-snapshot", () => {
  it("works...", () => {
    expect(() => checkSnapshot(<FixtureComponent />)).toThrow(iWasCalled());
  });
});
