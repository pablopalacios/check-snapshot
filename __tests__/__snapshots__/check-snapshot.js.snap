// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check-snapshot works... 1`] = `
<p>
  I'm a fixture component
</p>
`;
